Package.md

```diff
@@ -0,0 +1,189 @@
+The Azure Cosmos DB is Microsoft’s NoSQL database in the Azure technology stack. It is called a globally distributed 
```

Comment:
Check the comments in Module.md file. Most of them are applicable for this file as well it seems.

Reply:
Resolved same changes done as Module.md file